lib/sqlalchemy/sql/selectable.py

```diff
@@ -4347,24 +4375,24 @@ def _scalar_type(self) -> TypeEngine[Any]:
     def self_group(
         self, against: Optional[OperatorType] = None
     ) -> GroupedElement:
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325